Gordon360/Services/NewsService.cs

```diff
@@ -93,13 +110,14 @@ public StudentNews SubmitNews(StudentNews newsItem, string username, string id)
             var categoryIDParam = new SqlParameter("@CategoryID", newsItem.categoryID);
             var subjectParam = new SqlParameter("@Subject", newsItem.Subject);
             var bodyParam = new SqlParameter("@Body", newsItem.Body);
+            var imageParam = new SqlParameter("@Image", "");
 
             // Run stored procedure
-            var result = RawSqlQuery<StudentNewsViewModel>.query("INSERT_NEWS_ITEM @Username, @CategoryID, @Subject, @Body", usernameParam, categoryIDParam, subjectParam, bodyParam);
-            if (result == null)
-            {
-                throw new ResourceNotFoundException() { ExceptionMessage = "The data was not found." };
-            }
+            IEnumerable<int> idResult = _context.Database.SqlQuery<int>("INSERT_NEWS_ITEM @Username, @CategoryID, @Subject, @Body, @Image", usernameParam, categoryIDParam, subjectParam, bodyParam, imageParam);
```

Comment:
I think, since you're always uploading an empty imageParam, it's worthwhile to change `INSERT_NEWS_ITEM` back to not requiring an image param at all. There's no advantage to be gained in having it if all we ever do is put in an empty parameter, we should just let it be null until an actual image path is saved to the database.